integration/airflow/tests/integration/Dockerfile

```diff
@@ -34,11 +34,15 @@ RUN mkdir -p /app && mkdir -p /opt/openlineage
 RUN apt-get update && \
     apt-get install -y python3-dev default-libmysqlclient-dev build-essential
 COPY integration/common /opt/openlineage/integration/common
+COPY integration/sql /app/openlineage/integration/sql
+COPY integration/target /app/openlineage/integration/sql/target
 COPY integration/airflow /opt/openlineage/integration/airflow
 COPY client/python /opt/openlineage/client/python
 COPY integration/airflow/integration-requirements.txt /opt/openlineage/integration/integration-requirements.txt
 WORKDIR /app
 COPY integration/airflow/tests/integration/*.py ./
 COPY integration/airflow/tests/integration/pytest.ini ./
 COPY integration/airflow/tests/integration/requests requests
-RUN cd /opt/openlineage/integration/ && pip install --use-deprecated=legacy-resolver --user -r integration-requirements.txt
+RUN cd /app/openlineage/integration/sql && \
+    pip install openlineage-sql --no-index --find-links target/wheels --force-reinstall && \
+    cd /opt/openlineage/integration/ && pip install --use-deprecated=legacy-resolver --user -r integration-requirements.txt
```

Comment:
Not super familiar with the integration structure of these Dockerfiles, but shouldn't this be in the Great Expectations integration?

Reply:
Dependency change for common affects docker tests, and they need to be able to install locally-build package, because it's not actually on pypi 😉 
So all of this work is to build and install local deps - we want to test 100% code from local repo, and you can modify different parts of the monorepo at once.